docs/_docs/localization.md

```diff
@@ -90,6 +97,27 @@ to the node `Hello,` which you can rename in this step.
 
 The plugin also allows you to set a proper description and char limit for translations.
 
+#### 1.4.1 Break identical strings into different translations
+
+Below is an exmaple of the output strings with the group option checked.
+
+<img src="./localization/GroupedExample.png">
+
+The strings grouping several text nodes will have a break icon button for each of the node, by clicking on which,
+it will create a separate entry for that node with a different string resource id. The string resource id is auto
+generated by the plugin. You can rename it. Note, if you rename it to an exising string resource name with the
+same text, it will merge them together.
+
+<img src="./localization/UngroupedExample.png">
+
+The same, when it comes to the output with the group option unchecked, you can merge them by renaming them to use the same string resource name.
```

Comment:
I'm not sure what "The same" is referring to at the beginning of this sentence -- maybe leave it out, unless I'm misunderstanding.

Reply:
Done.